development/contribute-to-docs.md

```diff
@@ -0,0 +1,128 @@
+also need to create a GitHub account.
+
+Be sure to read the [Timescale docs contribution styleguide][timescale-docs-style]. 
+You’ll see information about how we refer to aspects of Timescale, 
+how we format our docs, and special Markdown tags available to 
+you as you author your contribution.
+
+Before we accept any contributions, Timescale contributors need to
+sign the [Contributor License Agreement][timescale-github-cla] (CLA).
+By signing a CLA, we can ensure that the community is free and confident in its
+ability to use your contributions.
+
+# Clone the documentation repository
+Timescale documentation is in the `docs.timescale.com-content` 
+[repository][timescale-docs-github]. You can clone this repository from 
+the command line like so:
+
+```bash
+git clone git@github.com:timescale/docs.timescale.com-content.git
+```
+
+Once cloned, you should be able to open the entire Timescale 
+documentation repository in a code editor of your choice.
+
+You can create a new branch in git as well:
+
+```bash
+git checkout -b my-branch-name
+```
+
+# Modifying a file
+
+Once you’re in your new branch you can start editing your doc as you normally would.
+
+At any time, you can see which files have been edited with the following git command:
+
+```bash
+git status
+```
+
+# Adding a new item to the page structure
+
+Often, you may want to add a new file to Timescale docs. To do so, you must 
+edit the `page-index/page-index.js` file. You should be able to glean 
+which lines to add and how to modify them based on the existing structure of 
+the file. If you need help, let us know in the 
+[Timescale community Slack][timescale-slack].
+
+# Submitting a pull request
+
+A pull request is a method of submitting contributions to an open development 
+project. It occurs when a developer asks for changes committed to an external 
+repository to be considered for inclusion in a project's main repository after 
+the peer review.
+
+Once you’ve completed all of your edits and are ready to submit a pull request, 
+you will first need to add all your edited files to your branch:
+
+```bash
+git add .
+```
+
+Now, you need to commit your changes along with a message explaining what changes 
+you’ve made to the repository.
+
+```bash
+git commit -m “a very brief explanation of what your pull request contains”
+```
+
+Then you have to upload your changes to GitHub:
+
+```bash
+git push origin my-branch-name
+```
+
+You will then need to complete the process on the 
+[Timescale Docs GitHub repository][timescale-docs-github]. Click on 
+the `New Pull Request` button and follow the instructions to create a 
+pull request using the branch you uploaded.
+
+# Making changes to a pull request
+
+Typically, you’ll be asked to make some changes to your pull request. You 
+can do so locally and re-add your changes to your branch and re-upload 
+it using the following commands:
+
+```bash
+git add .
+git commit --amend
+git push -f origin my-branch-name
```

Comment:
You might want to add the caveat that `git amend`ing and force pushing are our preferred method here b/c this is content and not code (otherwise we'd encourage adding commits and squashing down commits right before merging)